content/jobs/tanks/dark-knight/advanced-guide.md

```diff
@@ -25,13 +27,20 @@ However, it is often your only option for a GCD that can be used outside of mele
 
 In more advanced settings, we can be more nuanced --- for small amounts of GCD downtime, it may not always be correct to keep the GCD rolling with Unmend.
 DRK's Souleater combo has an average potency per GCD of 387, even before counting the resource gain from Syphon Strike and Souleater.
-If we have a downtime of `x` seconds (where `x` is smaller than 2.50 --- at 2.50 seconds or longer of downtime, Unmend can be freely used without delaying your next melee GCD), using Unmend gives 150 potency, but delays the next GCD by `(2.50 - x)` seconds, relative to if it had not been used.
-On average, this means that using Unmend costs `(2.50 - x)/([GCD speed])` of an average filler GCD, or `378 * (2.50 - x)/([GCD speed])`.
-The average potency gain/loss from using Unmend when there are `x` seconds remaining of downtime can then be written as `150 - (378 * (2.50 - x)/([GCD speed]))`.
+If we have a downtime of \(x\) seconds (where \(x\) is smaller than 2.50 --- at 2.50 seconds or longer of downtime, Unmend can be freely used without delaying your next melee GCD), using Unmend gives 150 potency, but delays the next GCD by \(2.50 - x\) seconds, relative to if it had not been used.
+On average, this means that using Unmend costs \(\frac{2.50 - x}{\rm{GCD Speed}}\) of an average filler GCD, or \(378 - \frac{2.50 - x}{\rm{GCD Speed}}\).
+The average potency gain/loss from using Unmend when there are \(x\) seconds remaining of downtime can then be written as \(150 - (378 \times \frac{2.50 - x}{\rm{GCD Speed}})\).
 When this quantity is positive, you gain damage from using Unmend, and when it is negative, you lose damage from using Unmend.
 
-Setting this equal to zero and solving for `x` in terms of GCD speed, we see that the break-even point for using Unmend is when downtime is `2.50 - (150/378 * ([GCD speed]))` seconds long.
-At a 2.50 GCD, this value is just over 1.5, and so using Unmend is a gain (on average) for downtime longer than 1.5 seconds, and a loss for downtime shorter than 1.5 seconds.
+Setting this equal to zero and solving for \(x\) in terms of GCD speed, we see that the amount of seconds break-even point for using Unmend is when downtime is the the result of the following equation:
+
+\[
+\begin{aligned}
+2.5 - \left(\frac{150}{378} \times \rm{GCD Speed} \right)
+\end{aligned}
+\]
+
+At a 2.50 GCD, this value is just over \(1.5\), and so using Unmend is a gain (on average) for downtime longer than 1.5 seconds, and a loss for downtime shorter than 1.5 seconds.
```

Comment:
I don't know if `\begin{equation}` will work here, but if it does, we could in principle replace "this value" with something like "the value of expression (1)". Whether that's clearer or not, I'm not entirely sure, but it may be more relevant for equations/expressions that we refer back to more often (as the guide gets built out more).